frontend/src/features/missions/MissionForm/GeneralInformationsForm.tsx

```diff
@@ -55,14 +60,22 @@ export function GeneralInformationsForm() {
         values.missionTypes && values.missionTypes.map(missionType => missionTypeEnum[missionType].libelle).join(' / ')
       } – ${values.controlUnits.map(controlUnit => controlUnit.name?.replace('(historique)', '')).join(', ')}`
 
+  const getStatus = () => {
+    if (isClosedMission) {
+      return <StatusTag bgColor={COLORS.opal}>{missionStatusLabels.CLOSED.libelle}</StatusTag>
+    }
+
+    return <MissionStatusTag status={getMissionStatus({ ...values, isClosed: false })} />
```

Comment:
J'ai l'impression qu'on peut directement utiliser MissionStatusTag
```suggestion
    return <MissionStatusTag status={getMissionStatus({ ...values, isClosed: isClosedMission  || false })} />
```